packages/@glimmer/runtime/lib/compiled/opcodes/expressions.ts

```diff
@@ -316,3 +316,29 @@ APPEND_OPCODES.add(Op.Log, (vm) => {
     })
   );
 });
+
+APPEND_OPCODES.add(Op.Equal, (vm) => {
+  let { positional } = check(vm.stack.pop(), CheckArguments).capture();
+  let stack = vm.stack;
+
+  stack.push(
+    createComputeRef(() => {
+      let values = reifyPositional(positional);
+
+      return values.every((v) => v === values[0]);
```

Comment:
I don't think we can use `.every` (IE11 doesn't have it), additionally we only need to support comparing two positionals.

Reply:
IE does have `.every` since version 9:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/every#browser_compatibility